packages/web-tracing/package.json

```diff
@@ -53,17 +53,17 @@
   },
   "dependencies": {
     "@grafana/faro-web-sdk": "^1.7.3",
-    "@opentelemetry/api": "^1.7.0",
-    "@opentelemetry/context-zone": "^1.18.1",
-    "@opentelemetry/core": "^1.18.1",
-    "@opentelemetry/exporter-trace-otlp-http": "^0.48.0",
-    "@opentelemetry/instrumentation": "^0.48.0",
-    "@opentelemetry/instrumentation-fetch": "^0.48.0",
-    "@opentelemetry/instrumentation-xml-http-request": "^0.48.0",
-    "@opentelemetry/otlp-transformer": "^0.48.0",
-    "@opentelemetry/resources": "^1.18.1",
-    "@opentelemetry/sdk-trace-web": "^1.18.1",
-    "@opentelemetry/semantic-conventions": "^1.18.1"
+    "@opentelemetry/api": "^1.9.0",
+    "@opentelemetry/context-zone": "1.21.0",
```

Comment:
Need to pin the zone version because all version above lead to issues.
It's bad practice but till we found the problem and a solution this should work.

Reply:
So, you are saying that  till `1.21.0` is fine any anything above is causing issue ?
I see the latest is `1.25.0`.

Reply:
Yes exactly. I don't know why yet.